src/main/java/net/sf/jabref/logic/util/OS.java

```diff
@@ -11,4 +11,24 @@
     public static final boolean LINUX = osName.startsWith("linux");
     public static final boolean WINDOWS = osName.startsWith("win");
     public static final boolean OS_X = osName.startsWith("mac");
+    
+    public static final String guessProgramPath(String programName, String windowsDirectory) {
+        if (OS.WINDOWS) {
+            String progFiles = System.getenv("ProgramFiles(x86)");
+            if (progFiles == null) {
+                progFiles = System.getenv("ProgramFiles");
+            }
+            if (windowsDirectory != null) {
+                return progFiles + "\\" + windowsDirectory + "\\" + programName + ".exe";
+            } else {
+                return progFiles + "\\" + programName + ".exe";
+            }
+        } else {
+            return programName;
+        }
+    }
+
+    public static final String guessProgramPath(String programName) {
+        return guessProgramPath(programName);
```

Comment:
Yes, that is a bug, should be return guessProgramPath(programName, null). Just a convenience method to avoid having to provide a path in case there isn't a Windows version of the program (in which case maybe one doesn't have to guess, but assuming that guessProgramPath is eventually improved and more universal).